src/utils/__tests__/getEntryFiles.test.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ *
+ * @flow
+ */
+
+const getEntryFiles = require('../getEntryFiles');
+const path = require('path');
+
+const fakePolyfill = 'polyfill/to/remove';
+const fakeCwd = '/usr/fake';
+
+beforeEach(() => {
+  // $FlowFixMe
+  process.cwd = () => fakeCwd;
```

Comment:
Can you reset `cwd` in `afterEach` hook?

Reply:
how can I do that? :D

Reply:
```js
let originalCwd = process.cwd;

beforeEach(() => process.cwd = jest.fn(() => fakeCwd));
afterEach(() => process.cwd = originalCwd);
```